examples/backends/vllm/launch/disagg_router_gaudi.sh

```diff
@@ -16,6 +16,8 @@ PT_HPU_LAZY_MODE=0
 NIXL_BUFFER_DEVICE=cpu
 VLLM_NIXL_BACKEND=UCX
 
+# Explicitly set NATS server for KV event publishing
+export NATS_SERVER="nats://localhost:4222"
```

Comment:
for tests to work correctly - we'll need to only set this when its not already set -

`
export MYVAR=${MYVAR:-<default value here>}
`